pkg/importer/data-processor.go

```diff
@@ -372,11 +377,11 @@ func (dp *DataProcessor) getUsableSpace() int64 {
 
 // GetUsableSpace calculates space to use taking file system overhead into account
 func GetUsableSpace(filesystemOverhead float64, availableSpace int64) int64 {
-	spaceWithOverhead := int64((1 - filesystemOverhead) * float64(availableSpace))
+	// +1 always rounds up.
+	spaceWithOverhead := int64((1 - filesystemOverhead) * float64(availableSpace+1))
```

Comment:
What if it's already 123234.0?

Reply:
IIRC the easiest way to do this was to just add one. but I can do a int64(math.Ceil(calculation)) instead, not sure if it is any better. I have the cases where it is even covered in my unit testing.